examples/cart/index.html

```diff
@@ -61,6 +61,14 @@ <h2 class="cart-title">Your cart</h2>
   <!-- .cart end -->
 
 
+  <!-- .cart-tab start -->
+  <button class="btn btn--cart-tab">
+    <div class="btn__counter"></div>
```

Comment:
Technically only phrasing content can live inside a `button`, which means this would need to be a `span` instead of a `div`. I’ve never seen it cause an actual problem, but if switching to `span` with `display: block` doesn’t cause issues it may be safer.